website/static/js/accountSettings.js

```diff
@@ -102,15 +102,20 @@ var UserProfileClient = oop.defclass({
         ).done(function (data) {
             ret.resolve(this.unserialize(data, profile));
         }.bind(this)).fail(function(xhr, status, error) {
-            $osf.growl('Error', 'User profile not updated. Please refresh the page and try ' +
+            if (error.toLowerCase() !== 'bad request') {
```

Comment:
Better to compare against the status code rather than the message. You should be able to get it from the `xhr` object